storybook/stories/upload.stories.js

```diff
@@ -145,6 +145,24 @@ storiesOf('Components|Upload', module)
       />
     </div>
   ))
+  .add('disabled button', () => (
+    <div className="py-3">
+      <Upload
+        clientId="a"
+        bucketId="b"
+        customerId="c"
+        multiple={boolean('Multiple File Select', Upload.defaultProps.multiple)}
+        max={number('Max number of files', 0)}
+        allowedFileTypes={array(
+          'Allowed File Types',
+          ['.jpg', '.jpeg', '.doc', '.docx'],
+          ','
+        )}
+        isDisabled
```

Comment:
rather than add a new story, let's add to the existing ones this prop:

```
isDisabled={boolean('Is Disabled', Upload.defaultProps.isDisabled)}
```

That way the developer looking at the story can toggle whether the input is disabled and see how the component behaves